src/main/kotlin/com/autonomousapps/subplugin/ProjectPlugin.kt

```diff
@@ -833,7 +833,8 @@ internal class ProjectPlugin(private val project: Project) {
     // Lists all import declarations in the source of the current project.
     val explodeCodeSourceTask = tasks.register<CodeSourceExploderTask>("explodeCodeSource$taskNameSuffix") {
       groovySourceFiles.setFrom(dependencyAnalyzer.groovySourceFiles)
-      dependencyAnalyzer.javaSourceFiles?.let { javaSourceFiles.setFrom(it) }
+      // dependencyAnalyzer.javaSourceFiles?.let { javaSourceFiles.setFrom(it) }
+      javaSourceFiles.setFrom(dependencyAnalyzer.javaSourceFiles)
```

Comment:
Second bug was treating Kotlin projects like they had no Java source code in a `src/<foo>/java` dir.